lib/features/family/features/login/cubit/family_login_cubit.dart

```diff
@@ -37,27 +37,38 @@ class FamilyLoginCubit extends CommonCubit<String, FamilyLoginSheetCustom> {
 
     try {
       await _authRepository.login(email, password);
-
       emitCustom(const FamilyLoginSheetCustom.successRedirect());
+      return;
     } catch (e, stackTrace) {
-      emitInitial();
+      emitData(email);
+
+      final errorDialogs = {
```

Comment:
I would make it a field outside of the method. Now everytime you get into the catch method this gets reassigned while it never changes which is inefficient.